WordPress/gutenberg
# Fonts Library: change upload directory to wp-content/fonts

**What**
This PR closes https://github.com/WordPress/gutenberg/issues/53965.

**How**
We can't use `wp_handle_upload` because this function only works to move files into the `uploads` directory (or subdirectory). This PR reworks the implementation to `rename` the temporary file to its new location in the `wp-content/fonts` directory.

**To test**
- Ensure the related `wpFontFamily` tests pass 
- Check out https://github.com/WordPress/gutenberg/pull/53884 and verify you can install / uninstall a font successfully.
